File: Tutorials/cdk-hello-world-template/python/hello_cdk/hello_cdk_stack.py
from aws_cdk import (
    Stack,
    CfnOutput,
    aws_lambda as _lambda,
)
from constructs import Construct

class AcmeServerlessStack(Stack):

    def __init__(self, scope: Construct, construct_id: str, **kwargs) -> None:
        super().__init__(scope, construct_id, **kwargs)

        # Define the Lambda function resource
        my_function = _lambda.Function(
            self, "AcmeApiFunction",
            runtime=_lambda.Runtime.NODEJS_18_X,  # Provide any supported Node.js runtime
            handler="index.handler",
            code=_lambda.Code.from_inline(
                """
                exports.handler = async function(event) {
                  return {
                    statusCode: 200,
                    body: JSON.stringify('Welcome to Acme Corp API!'),
                  };
                };
                """
            ),
        )

        # Define the Lambda function URL resource
        my_function_url = my_function.add_function_url(
            auth_type=_lambda.FunctionUrlAuthType.NONE,
        )

        # Define a CloudFormation output for your URL
        CfnOutput(self, "AcmeFunctionUrlOutput", value=my_function_url.url)

File: Tutorials/cdk-hello-world-template/csharp/src/HelloCdk/HelloCdkStack.cs
using Amazon.CDK;
using Constructs;
// Import the Lambda module
using Amazon.CDK.AWS.Lambda;

namespace AcmeServerlessApp
{
    public class AcmeServerlessStack : Stack
    {
        internal HelloCdkStack(Construct scope, string id, IStackProps props = null) : base(scope, id, props)
        {
            // Define the Lambda function resource
            var myFunction = new Function(this, "AcmeApiFunction", new FunctionProps
            {
                Runtime = Runtime.NODEJS_18_X, // Provide any supported Node.js runtime
                Handler = "index.handler",
                Code = Code.FromInline(@"
                    exports.handler = async function(event) {
                        return {
                            statusCode: 200,
                            body: JSON.stringify('Welcome to Acme Corp API!'),
                        };
                    };
                "),
            });

            // Define the Lambda function URL resource
            var myFunctionUrl = myFunction.AddFunctionUrl(new FunctionUrlOptions
            {
                AuthType = FunctionUrlAuthType.NONE
            });

            // Define a CloudFormation output for your URL
            new CfnOutput(this, "AcmeFunctionUrlOutput", new CfnOutputProps
            {
                Value = myFunctionUrl.Url
            });
        }
    }
}